doc/overview.rst

```diff
@@ -1550,11 +1550,15 @@ them to your local environment. This is a great way to test your deep learning s
 managed training or hosting environments. Local Mode is supported for frameworks images (TensorFlow, MXNet, Chainer, PyTorch,
 and Scikit-Learn) and images you supply yourself.
 
-You can install all necessary for this feature dependencies using pip:
+You can install necessary dependencies for this feature using pip; local mode also requires docker-compose which you can
+install using curl. (These instructions are for Linux. For Windows/MacOS docker-compose is installed with Docker Desktop.
```

Comment:
wording improvement: `which you can install using following steps.` [dont mention curl].
Also I think the curl instructions should also work for MacOS.